horizons/component/depositcomponent.py

```diff
@@ -44,11 +44,11 @@ def initialize(self, inventory=None):
 	def get_res_ranges(self):
 		"""Generator for tuples (res_id, min, max) for each resource that the deposit
 		can contain (as defined in the object file)."""
-		return ( (res, data.get('min_amount', 0), data['max_amount'])
-		         for res, data in self.resources.iteritems() )
+		return ((res, data.get('min_amount', 0), data['max_amount'])
+		        for res, data in self.resources.iteritems())
 
 	def get_random_res_amounts(self):
 		"""Generator for tuples (res_id, rand_amount) for each resource that the deposit
 		can contain (as defined in the object file)."""
-		return ( (res, self.session.random.randint(min_amount, max_amount))
-		         for res, min_amount, max_amount in self.get_res_ranges() )
+		return ((res, self.session.random.randint(min_amount, max_amount))
+		        for res, min_amount, max_amount in self.get_res_ranges())
```

Comment:
So now I have to wonder, do you truly think the two lines are more readable this way?